Staging/EnableGop/EnableGop.inf

```diff
@@ -12,7 +12,7 @@
   BASE_NAME                      = EnableGop
   FILE_GUID                      = 3FBA58B1-F8C0-41BC-ACD8-253043A3A17F
   MODULE_TYPE                    = DXE_DRIVER
-  VERSION_STRING                 = 1.1
+  VERSION_STRING                 = 1.2
```

Comment:
NB I am not planning to update this with every release, only if the features change significantly (or if the image size for some other reason changed very significantly).